.github/workflows/merchant-sdk.publish.example.apps.firebase.yml

```diff
@@ -1,5 +1,10 @@
 name: Publish GiniMerchantSDKExample to Firebase Distribution
 on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - 'MerchantSDK/**'
+      - 'HealthAPILibrary/**'
```

Comment:
[Suggestion]: @ValentinaIancu-Gini  we need to add here `GiniInternalPaymentSDK` and `GiniUtilites`